src/openrct2/scripting/bindings/world/ScTileElement.cpp

```diff
@@ -1049,35 +1089,37 @@ namespace OpenRCT2::Scripting
             duk_push_int(ctx, idx.ToUnderlying());
         return DukValue::take_from_stack(ctx);
     }
-    void ScTileElement::bannerIndex_set(uint16_t value)
+    void ScTileElement::bannerIndex_set(const DukValue& value)
     {
-        ThrowIfGameStateNotMutable();
-
-        switch (_element->GetType())
+        if (value.type() == DukValue::Type::NUMBER)
```

Comment:
Perhaps allow banner index to be set to null if the `null` is passed in?

Reply:
Yes that makes sense.